source/developer/interactive-message-buttons.rst

```diff
@@ -1,209 +1,4 @@
-.. _interactive-message-buttons:
-
 Interactive Message Buttons
 ============================
 
-Mattermost supports interactive message buttons for `incoming <https://docs.mattermost.com/developer/webhooks-incoming.html>`_ and `outgoing webhooks <https://docs.mattermost.com/developer/webhooks-outgoing.html>`_, and for `custom slash commands <https://docs.mattermost.com/developer/slash-commands.html>`_ via actions. They help make your integrations richer by completing common tasks inside Mattermost conversations, increasing user engagement and productivity.
-
-Use these buttons to simplify complex workflows by allowing users to take quick actions directly through your integration post. For example, the buttons enable your integration to:
-
-- update sales opportunities in your CRM system from Mattermost
```

Comment:
Wondering if we would want to consider use cases that are a little more compelling to a developer audience?  0/5 although I would anticipate they would test this feature by adding something useful to their workflow first. Maybe consider "mark a task complete in your project management tracker from Mattermost" or "initiate a command from Mattermost to merge a branch into a release".